superset/reports/commands/exceptions.py

```diff
@@ -258,6 +261,17 @@ class ReportScheduleStateNotFoundError(CommandException):
     message = _("Report Schedule state not found")
 
 
+class ReportScheduleSystemErrorsException(CommandException, SupersetErrorsException):
+    errors: List[SupersetError] = []
```

Comment:
Do we want to set an overall status for the array or do we want  to do it based upon the most critical error in the error array

Reply:
we set the status based upon the most critical